pallets/dapps-staking/src/testing_utils.rs

```diff
@@ -18,17 +18,17 @@
 
 use super::{pallet::pallet::Event, *};
 use frame_support::assert_ok;
-use mock::{EraIndex, *};
+use mock::{Balance, EraIndex, *};
```

Comment:
`Balance` is not used here either no?

Reply:
`Balance` is used, same issue, making it unambigious.